pages/etherpad/[[...roomId]].js

```diff
@@ -172,132 +175,11 @@ export default function Etherpad() {
         return room;
     }, [auth, matrix, matrixClient, etherpad]);
 
-    const ActionNewAnonymousPad = ({ callbackDone }) => {
```

Comment:
Not actually removing any of this code. These 4 functions were just moved to their own files, each.